### ISSUE TITLE
Business Plan

### ISSUE THREAD

**Issue by erlend-sh** [commune-org/weird#12]
I plan to write the Weird version of what I already wrote for a former employer:
https://blog.meilisearch.com/meilisearch-open-business-plan-v0-1/

This article explains my basic investor/builder thesis:
https://spicylobster.itch.io/jumpy/devlog/337996/everlasting-games

It’s the [Open Product Recipe](https://blog.erlend.sh/open-source-explained).
WordPress vs Movable Type
Discourse vs VBulletin
Mattermost vs Slack
Strapi vs Contentful
etc.

In the short term, the open approach is a Good Enough differentiator. In the long term, it’s an objectively better way to build.

So this is what we’re doing for weird-app as well. Starting with a linksapp as the smallest atomic unit upon which a much larger product and company can be built.

Weird’s parent company ethos will be the same as @spicylobstergames :
https://spicylobster.itch.io/jumpy/devlog/356201/spicy-lobster-open-gamedev-company

## Co-ownership & equity

Everyone I work with is a potential co-owner of whichever project were we’re doing together. I’ve no interest in being the majority owner of anything I build, since I’m not much of a solo-builder.

In fact, I don’t really think it’s good for a company/product to have any owner with more than 5% ownership, assuming it’s big enough to have 20+ people involved.

So even if let’s say me and one other founding dev built this company + product just the two of us together as 50/50 partners (which is already not the case, as we’ve got several collaborators involved ❤️‍🔥), my long-term intent would be to bring our total ownership % down to 5% over the next few years.

I also don’t think someone who’s not actively working in a company should get to sit on their share for very long until it starts automatically being sold off. I’m no fan of rent-seeking dynamics.

> @jbutterwick said:
> I've been imagining a software house as a worker-owned co-op based on some of ideas I've heard from Yanis Varoufakis at his talks for DiEM25, still finding my way of course, but it sounds like you've got some strong ideas and ideals around this
> This talk of his https://youtu.be/9aK4OztueuE?t=3861 has helped shape some of my ideas on the subject of organizational hierarchy and ownership
> I don't necessarily agree with absolutely everything about it but as a model I quite enjoy his proposal

## Investment

I’ve looked around at a lot investors to find some that align with my values, and I’ve found one clear favorite: https://calmfund.com

In short, they focus on taking a revenue split rather than hoping for some big exit. Also, they actually *require* that a company is earning about $5000/month before they invest, meaning they intentionally have very little leverage.

There are other great options out there like https://oss.capital (quite like YC specialized in open source & cloud), but Calm provides the best possible starting point.

Calm is Plan A, I.e. spending the next year trying to simply make a small and steady profit. What’s really cool about them is that they also don’t do the all-or-nothing pitch. Instead they invite founders to simply get in touch with them and keep an open line going.

So as soon as we’re making $50/month, I’ll start pitching to them.
